packages/kbn-apm-synthtrace/src/scenarios/mobile.ts

```diff
@@ -415,9 +407,7 @@ const scenario: Scenario<ApmFields> = async (runOptions: RunOptions) => {
         });
       };
 
-      return [...androidDevices, ...iOSDevices]
-        .map((device) => logger.perf('generating_apm_events', () => sessionTransactions(device)))
-        .reduce((p, c) => p.merge(c));
+      return [...androidDevices, ...iOSDevices].map((device) => sessionTransactions(device));
```

Comment:
I don't think it's particularly useful. The performance bottleneck will be in the transformation part which is not easily measurable because of streaming (things no longer happen in consecutive isolated chunks of work).